snap/local/start_mongod.sh

```diff
@@ -6,6 +6,11 @@ if [ ! -f "${CONFIG_FILE}" ]; then
         echo "copying default config to ${CONFIG_FILE}"
```

Comment:
nit: change the name of this folder to `snap/hooks/start_mongod.sh` for consistancy across DPE snaps